external-crates/move/crates/move-analyzer/tests/symbols/sources/M1.move

```diff
@@ -1,6 +1,6 @@
 module Symbols::M1 {
 
-    struct SomeStruct has key, drop, store {
```

Comment:
These changes are unrelated, but remove compilation errors which do not affect symbolication but are annoying when working with these Move files